lib/circlex/emulator/state.ex

```diff
@@ -243,6 +243,9 @@ defmodule Circlex.Emulator.State do
   defp generate_type(:external_ref),
     do: "EXTREF" <> to_string(System.unique_integer([:positive]))
 
+  defp generate_type(:virtual_account_number),
+    do: Enum.random(100_000_000_000..999_999_999_999) |> to_string()
```

Comment:
Just curious, so it will always generate a different virtual_account_number every time ?
So if from treasury app, we called get_wire_instruction with same bank id, will we get two different virtual account number or two identical account number ?

Reply:
Oh I see. only create when creating bank info. When instruction api is invoke it only retrieve the data.

👍  :P